### PR TITLE
Add the jaxws-2.3 api

### PR DESCRIPTION
**Pull request by neuwerk**

### CODE REVIEW

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.internal.jaxws-2.3.feature [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.hc.3.2
+-files=\
+bin/jaxws/wsgen; ibm.executable:=true; ibm.file.encoding:=ebcdic, \
+bin/jaxws/wsimport; ibm.executable:=true; ibm.file.encoding:=ebcdic, \
+dev/api/ibm/schema/ibm-ws-bnd_1_0.xsd, \
+bin/jaxws/wsimport.bat, \
+bin/jaxws/tools/ws-wsimport.jar, \
+bin/jaxws/wsgen.bat, \
+bin/jaxws/tools/ws-wsgen.jar
+kind=beta
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.internal.jaxws-2.3.feature]: how can this feature be beta when `com.ibm.websphere.appserver.jaxws-2.3` is noship?

File: dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.internal.jaxws-2.3.feature [hunk 2]
```diff
@@ -0,0 +1,50 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.websphere.appserver.internal.jaxws-2.3
+visibility=private
+singleton=true
+IBM-App-ForceRestart: uninstall, \
+install
+Subsystem-Name: Internal Java RESTful Services 2.1
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.internal.jaxws-2.3.feature]: Shouldn't that version be 2.3?

File: dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.3/resources/l10n/com.ibm.websphere.appserver.jaxws-2.3_ja.properties [hunk 3]
```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Copyright (c) 2012 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.3/resources/l10n/com.ibm.websphere.appserver.jaxws-2.3_ja.properties]: wrong date

File: dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.3/resources/l10n/com.ibm.websphere.appserver.jaxws-2.3_ja.properties [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=\u3053\u306e\u30d5\u30a3\u30fc\u30c1\u30e3\u30fc\u306f\u3001Java API for XML-Based Web \u30b5\u30fc\u30d3\u30b9 2.2 \u306e\u30b5\u30dd\u30fc\u30c8\u3092\u53ef\u80fd\u306b\u3057\u307e\u3059\u3002 \
```
Comment (tevans78) [inline dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.3/resources/l10n/com.ibm.websphere.appserver.jaxws-2.3_ja.properties]: wrong version

File: dev/com.ibm.websphere.javaee.jaxws.2.3/.settings/org.eclipse.jdt.core.prefs [hunk 5]
```diff
@@ -0,0 +1,276 @@
+#Ant properties
+#Automatically generated by the ant prepare.settings.files task
+org.eclipse.jdt.core.compiler.codegen.inlineJsrBytecode=enabled
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.7
+org.eclipse.jdt.core.compiler.compliance=1.7
```
Comment (tevans78) [inline dev/com.ibm.websphere.javaee.jaxws.2.3/.settings/org.eclipse.jdt.core.prefs]: 1.8

File: dev/com.ibm.websphere.javaee.jaxws.2.3/.classpath [hunk 6]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.7"/>
```
Comment (tevans78) [inline dev/com.ibm.websphere.javaee.jaxws.2.3/.classpath]: 1.8

File: dev/com.ibm.websphere.javaee.jaxws.2.3/bnd.bnd [hunk 7]
```diff
@@ -0,0 +1,26 @@
+#*******************************************************************************
+# Copyright (c) 2017 IBM Corporation and others.
```
Comment (tevans78) [inline dev/com.ibm.websphere.javaee.jaxws.2.3/bnd.bnd]: please check all these copyright dates

File: dev/com.ibm.ws.jaxws.2.3.common/bnd.bnd [hunk 8]
```diff
@@ -0,0 +1,222 @@
...
+    com.ibm.xml.xlxp2.api.wssec
+    
+
+-dsannotations: com.ibm.ws.jaxws.metadata.builder.AnnotationEndpointInfoConfigurator,\
+  com.ibm.ws.jaxws.metadata.builder.WebServicesBndEndpointInfoConfigurator,\
+  com.ibm.ws.jaxws.client.injection.ServiceRefObjectFactory,\
+  com.ibm.ws.jaxws.globalhandler.JaxwsGlobalHandlerServiceImpl
+  
+
+Service-Component: \
```
Comment (tevans78) [inline dev/com.ibm.ws.jaxws.2.3.common/bnd.bnd]: Are there any plans to switch these to be annotations in the code?

File: dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.3/resources/l10n/com.ibm.websphere.appserver.jaxws-2.3_ja.properties (hunk 4 above)
Comment (tevans78) [reply dev/com.ibm.websphere.appserver.features/visibility/public/jaxws-2.3/resources/l10n/com.ibm.websphere.appserver.jaxws-2.3_ja.properties]: with pre-translated files, you can't assume that the translators will spot errors or update copyright dates

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by neuwerk** [comment]
#build

**Comment by tevans78** [comment]
@neuwerk please squash your commits before merging. 22 commits is far too many. Thank you.

**Comment by neuwerk** [comment]
Closing as this pull request has conflicts with integration and it was easier to just create a new branch and new pull request.

**Pull request closed without merging**